docs/src/main/asciidoc/modules/ROOT/pages/sql2cypher.adoc

```diff
@@ -9,23 +9,18 @@ The translation from SQL queries to Cypher is an optional feature of this driver
 It consists of two interfaces, `SqlTranslatorFactory` and the actual `SqlTranslator`.
 - A concrete implementation of this SPI, published as `{group-id}:neo4j-jdbc-translator-impl`.
 
-The latter is bundled in the "full bundle" as described in xref:available_bundles[xrefstyle=short].
+The latter is covered in "xref:s2c[]" and bundled as the "full bundle", described in xref:available_bundles[xrefstyle=short].
 The former is provided for two reasons: It allows us to distribute the driver with and without the bundled, default translator and can be an option for you to run your custom translator.
 
+Translators can be chained, and you can have as many translators on the class-path as you want.
```

Comment:
```suggestion
Translators can be chained, and you can have as many translators on the class path as you want.
```

Also note that there is as well _classpath_ in the current documentation. This should get unified.

Reply:
Most official docs use `class path`

Reply:
Then my suggestion is of course
```suggestion
Translators can be chained, and you can have as many translators on the classpath as you want.
```
😺